loading.go

```diff
@@ -35,7 +36,13 @@ func LoadStrategy(path string, local, remote func(string) ([]byte, error)) func(
 }
 
 func loadHTTPBytes(path string) ([]byte, error) {
-	resp, err := http.Get(path)
+	client := new(http.Client)
+	client.Timeout = 30 * time.Second
```

Comment:
I think it should be configurable. 30 seconds may or may not be enough for some cases.
Let say some time 30 seconds response is FOREVER for services where response over 1 second is useless.

client := &http.Client{Timeout: 30 * time.Second} looks nicer to my eyes ;)